transcom/mymove
# INT B-20792 add gsr_appeals table

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20792)

## Summary

As we set up work to create the GSR office user, we need to create the table that will hold all of the appeal data for the backend and UI to use.

This PR does the following:
- creates migration file to create the `gsr_appeals` table and associated columns
- adds a Go struct to accommodate the new table
- adds tests

### How to test

1. Fire up your `server` and build
2. Open up DBeaver and confirm you see the table with all associated values and configuration

## Screenshots
![Screenshot 2024-08-12 at 2 43 20 PM](https://github.com/user-attachments/assets/da6db2a7-da8a-4a0a-8d91-92852c5533c7)
